api/src/accounts.rs

```diff
@@ -218,6 +218,28 @@ impl Account {
         })
     }
 
+    // Return the latest ledger info from internal indexer
+    pub fn new_with_internal_indexer(
+        context: Arc<Context>,
+        address: Address,
+        start: Option<StateKey>,
+        limit: Option<u16>,
+    ) -> Result<Self, BasicErrorWith404> {
+        match context.get_latest_internal_indexer_ledger_version_and_info::<BasicErrorWith404>() {
+            Ok((latest_internal_indexer_ledger_info, latest_internal_indexer_ledger_version)) => {
+                Ok(Self {
+                    context,
+                    address,
+                    ledger_version: latest_internal_indexer_ledger_version,
+                    start,
+                    limit,
+                    latest_ledger_info: latest_internal_indexer_ledger_info,
+                })
+            },
+            Err(_) => Account::new(context, address, None, start, limit),
```

Comment:
I'm not comfortable treating all errors here as if the internal indexer doesn't exist -- the context knows perfectly if it's present, right? why relying on the (generic) error?